src/Tools/ExternalAccess/FSharp/Navigation/IFSharpDocumentNavigationService.cs

```diff
@@ -4,42 +4,38 @@
 
 #nullable disable
 
+using System.Threading;
 using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Navigation;
 using Microsoft.CodeAnalysis.Options;
 using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.FSharp.Navigation
 {
     internal interface IFSharpDocumentNavigationService : IWorkspaceService
```

Comment:
@cartermp Adding new overloads that take cancellation tokens. Existing overloads will remain for binary compat. Once you can update your code over to the new overloads, we can remove the old ones.

Reply:
Sounds good. We'll have to do some work to plumb some options through for Inline Hints (pushed out to 16.10) so I'll use that as an opportunity to update dependencies and use the new overload.

Reply:
Sounds good :)

Reply:
Do we have a bug tracking the cleanup so we can know when it's safe to remove this?

Reply:
No.  In general, i don't find that a useful mechanism to effect change.  Better to just ask for the chagne to happen and remove once it does.